Gruntfile.js

```diff
@@ -30,13 +30,22 @@ module.exports = function(grunt) {
     'webpack:functional_tests',
   ]);
 
-  grunt.registerTask('test', [
-    'lint',
-    'flowbin:check',
-    'build-tests',
-    'mochaTest:unit',
-    'mochaTest:functional',
-  ]);
+  grunt.registerTask('test', 'run linting and test suites', function() {
+    var tasks = [
+      'lint',
+      'flowbin:check',
+      'build-tests',
+      'mochaTest:unit',
+      'mochaTest:functional',
+    ];
+
+    if (process.env.APPVEYOR) {
+      tasks = tasks.filter((t) => t !== 'flowbin:check');
```

Comment:
sure, I opened #773 and #774 (and added a reference to them in an inline comment near the skipped grunt task and the skipped testcase)